src/renderer/store/transcriptionWords/ops/correctWord.ts

```diff
@@ -21,7 +21,7 @@ export const makeCorrectWord: (
       selectionRangeAdded(rangeLengthOne(index)),
     ],
     undo: [
-      undoWordCorrected(index, prevText, prevConfidence ?? 1),
+      undoWordCorrected(index, prevText as string, prevConfidence ?? 1),
```

Comment:
Is this just for clarity? Or was the typing not working properly?

Reply:
action type assumes text to be a string but with the addition of pauses text can be null, hence the cast as we know that if a word is being corrected it must not have been a pause since you can't correct pauses

Reply:
Ah yep that makes sense!